c2corg_ui/static/js/suggestiondirective.js

```diff
@@ -0,0 +1,54 @@
+goog.provide('app.suggestionDirective');
+
+/**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$sce} $sce Angular Strict Contextual Escaping service.
+ * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
+ */
+app.suggestionDirective = function($compile, $sce) {
+  var html = '';
+  var header = '<span class="suggestion-text">' +
+                            '<span class="suggestion-title">' +
+                              '<span ng-bind-html="highlight(label, query)"></span>' +
+                              '<span ng-if="date_end">&nbsp;({{::date_end }})</span>' +
+                            '</span><br>' +
+                            '<span class="suggestion-info">' +
+                            '  <span ng-if="elevation">{{::elevation}}m</span>' +
+                            '</span>';
+  var footer =   '</span>';
```

Comment:
This "generic" header looks weird: we have no ``date_end`` or ``elevation`` for all document types.
What about testing the doctype first and then call functions dedicated to a given doctype that would return the corresponding template?